src/assisted_test_infra/download_logs/__main__.py

```diff
@@ -89,6 +99,11 @@ def handle_arguments():
         type=str,
         default=os.path.join(os.getenv("HOME"), ".kube/config"),
     )
+    parser.add_argument(
+        "--clusters-to-filter-out-file",
```

Comment:
maybe `--exclude-clusters` or `--exclude-installs` would be more descriptive

Reply:
Done, thanks!